openrcv3_domain

```diff
@@ -12,7 +12,7 @@ else
 fi
 _password=$(juju run $_juju_model_arg --unit keystone/leader 'leader-get admin_passwd')
 
-if [ -s $_root_ca ]; then
+if [ ! -s $_root_ca ]; then
```

Comment:
Can you explain the rationale behind this change? The new code reads like "if the file _root_ca doesn't exist, let's use https" and that doesn't make sense to me. What am I getting wrong? Thanks!

https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

Reply:
ah I understand now, what you want is "if string not empty and if file exists", correct?